gcp/appengine/deploy.sh

```diff
@@ -21,17 +24,14 @@ for config in $configs_dir/*.yaml; do
 done
 
 cd "$dir"
+image="gcr.io/$project_id/osv-website"
+
+if [ -z "$CLOUDBUILD" ]; then
```

Comment:
```suggestion
if [[ -z "$CLOUDBUILD" ]]; then
```
Adhere to https://google.github.io/styleguide/shellguide.html#test----and---